src/at_commands/at_commands.rs

```diff
@@ -40,7 +37,7 @@ impl AtCommandsContext {
     pub async fn new(
         global_context: Arc<ARwLock<GlobalContext>>,
         n_ctx: usize,
-        top_n: usize,
+        top_n: Option<usize>,
```

Comment:
what is the purpose of this change?

Reply:
not always top_n is needed. This top_n applies to quantity of postprocessing results as well. For cat command I need no such limit, thus I made it optional. @olegklimov has complains about this change too. Maybe, I just need to set a higher limit..